obs-packaging/scripts/obs-pkgs.sh

```diff
@@ -9,7 +9,7 @@
 OBS_PKGS_PROJECTS=(
 	qemu-lite
 	qemu-vanilla
-	kernel
+	linux-container
```

Comment:
Have you tried running this locally? I am afraid it will break this: https://github.com/kata-containers/packaging/blob/01b68b1ca65969db6406eeb74b5fc92ffbe33b2e/obs-packaging/build_all.sh#L45-L50

This is also discussed here #200

Reply:
I just tested, it works he also changed the directory name. 
I used the current branch to push to https://build.opensuse.org/project/show/home:katacontainers:release

All the build worked. And tested kata-runtime with ubuntu :)

Reply:
Oh, So #200 got fixed here? I was working on @marcov suggestion on an associative array and testing my patch out.